ui.spec

```diff
@@ -1,116 +1,59 @@
-%if %{?rhel}%{!?rhel:0} >= 7
-%global el6 0
-%else
-%global el6 1
-%endif
-Name:		ui
-Version:	4.0.3
-Release:	1%{?dist}
-Summary:	UI meta-packages
-Group:		Applications/Internet
-License:	ASL 2.0
-BuildRoot:	%(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
+Name: ui
+Version: 4.0.3
+Release: 1%{?dist}
+Summary: UI meta-packages
+Group: Applications/Internet
+License: ASL 2.0
+BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 
 # the above replaced by ca-policy-egi-core
-Requires:       ca-policy-egi-core
-Requires:       aria2
-Requires:	      davix-libs
-Requires:       dcache-srmclient
-Requires:       dcap
-Requires:	      dcap-devel
-Requires:	      dcap-libs 
-Requires:	      dcap-tunnel-gsi
-Requires:	      dcap-tunnel-krb  
-Requires:	      dcap-tunnel-ssl  
-Requires:	      dcap-tunnel-telnet
-Requires:       dpm
-Requires:       dpm-devel
-Requires:       dpm-perl
-Requires:       dpm-python
-Requires:       lcgdm-devel
-Requires:       libdpm.so.1()(64bit), dpm-libs
-Requires:       liblcgdm.so.1()(64bit), lcgdm-libs
-Requires:	      fetch-crl
-Requires:	      fts-client
-Requires:     	 fuse
-Requires:       fuse-libs
-Requires:       gfal2-all
-Requires:	      gfal2-plugin-xrootd
-Requires:       gfal2-python
-Requires:	      gfal2-util
-Requires:       gfalFS
-Requires:	      gfal2-doc
-Requires:	      gfal2-devel
-Requires:	      ginfo
-Requires:       glite-ce-cream-cli  
-Requires:       glite-ce-cream-client-api-c
-Requires:	      glite-yaim-core
-Requires:	      glite-yaim-clients
-Requires:	      gsi-openssh-clients
-Requires:	      globus-gsi-cert-utils-progs
-Requires:       lcg-ManageVOTag  
-Requires:       lcg-info  
-Requires:       lcg-infosites  
-Requires:       lcg-tags  
-Requires:       lfc
-Requires:       liblfc.so.1()(64bit), lfc-libs
-Requires:       lfc-devel
-Requires:       lfc-perl
-Requires:       lfc-python
-Requires:	      myproxy
-Requires:       nordugrid-arc-client
-Requires:       nordugrid-arc-plugins-xrootd
-Requires:       nordugrid-arc-plugins-gfal
-Requires:	      nordugrid-arc-plugins-globus
-Requires:	      nordugrid-arc-plugins-needed
-Requires:       openldap-clients
-Requires:       voms  
-Requires:       voms-clients-java  
-Requires:	      xrootd-client
+Requires: ca-policy-egi-core
+Requires: aria2
+Requires: condor
+Requires: davix-libs
+Requires: dcache-srmclient
+Requires: dcap
+Requires: dcap-devel
+Requires: dcap-libs
+Requires: dcap-tunnel-gsi
+Requires: dcap-tunnel-krb
+Requires: dcap-tunnel-ssl
+Requires: dcap-tunnel-telnet
+Requires: fetch-crl
+Requires: fts-client
+Requires: fuse
+Requires: fuse-libs
+Requires: gfal2-all
+Requires: gfal2-plugin-xrootd
```

Comment:
i would also add gfal2-plugin-http

Reply:
So in fact `gfal2-all` may already contain all the plugins, no?
cf https://github.com/cern-fts/gfal2/blob/develop/packaging/rpm/gfal2.spec#L187

Reply:
So I would basically only keep things that may not be included by gfal2-all

Reply:
yes true:-) so all the plugins are there, the other packages already there have to be installed separately